openedx/core/lib/api/paginators.py

```diff
@@ -3,6 +3,31 @@
 from django.http import Http404
 from django.core.paginator import Paginator, InvalidPage
 
+from rest_framework.response import Response
+from rest_framework import pagination
+
+
+class DefaultPagination(pagination.PageNumberPagination):
```

Comment:
To control the scope of this PR, I'd really prefer to avoid changing the APIs.  Could we standardize the pagination and update the clients as follow-on work?